registry/domain/models/service.py

```diff
@@ -1,6 +1,6 @@
 class Service:
     def __init__(self, org_uuid, uuid, service_id, display_name, short_description, description, project_url, proto,
-                 assets, ranking, rating, contributors, metadata_ipfs_hash, groups):
+                 assets, ranking, rating, contributors, metadata_ipfs_hash, groups, state):
```

Comment:
Please introduce one entity containing state information (state, created_at, updated_at, created_by, updated_by) in domain layer.